src/core/ext/filters/load_reporting/server_load_reporting_filter.cc

```diff
@@ -30,7 +30,13 @@
 #include "src/core/lib/channel/channel_args.h"
 #include "src/core/lib/channel/context.h"
 #include "src/core/lib/iomgr/resolve_address.h"
+
+#ifdef WIN32
+#include "src/core/lib/iomgr/sockaddr_windows.h"
+#else
 #include "src/core/lib/iomgr/sockaddr_posix.h"
+#endif
+
```

Comment:
the file `sockaddr.h` actually provides this switch already. You can avoid the whole logic here by including this other file instead.

Reply:
Also, each of these files have their own guards, so it's also okay to simply include them both.